style.css

```diff
@@ -0,0 +1,36 @@
+    *{
+    margin : 0;
+    padding : 0;
+    box-sizing : border-box;
+    font-family : 'Poppins', sans-serif;
+}
+.hero{
+    background:#1d2026;
```

Comment:
use a space after the property name's colon.

Here is the Google style guide for explanation: https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops